src/hotspot/share/cds/cds_globals.hpp

```diff
@@ -127,6 +127,9 @@
   product(bool, AOTCacheParallelRelocation, true, DIAGNOSTIC,               \
           "Use parallel relocation code to speed up startup.")              \
                                                                             \
+  product(bool, VerifyTrainingData, trueInDebug, DIAGNOSTIC,                \
+          "Verify archived training data")                                  \
+                                                                            \
```

Comment:
Should this flag be in compiler_globals.hpp?

Reply:
Yes, agreed.